src/ontology/edits/oeo-physical.omn

```diff
@@ -6773,8 +6902,14 @@ Class: OEO_00020038
     Annotations: 
         OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
         <http://purl.obolibrary.org/obo/IAO_0000115> "Solar radiation is radiation that is emitted by the sun.",
+        <http://purl.obolibrary.org/obo/IAO_0000116> "Solar radiation is the merological sum of direct solar radiation and diffuse solar radiation",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000116> "Solar radiation is the merological sum of direct solar radiation and diffuse solar radiation",
```
Is "merological sum" a typo or something that exists?

Reply:
Its a typo, it should be **mereological**.